### DISCUSSION TITLE
Recommended naming convention should be changed or mapStores should be updated

### DISCUSSION THREAD

**Discussion by jonatan-mobal** [vuejs/pinia#1578] [Ideas]
[The recommended naming convention](https://pinia.vuejs.org/cookbook/migration-vuex.html#restructuring-modules-to-stores) for nested stores is, for example, 'nested/module'.
But now, if you want to use the `mapStores` API, the default mapping will be to `nested/moduleStore`. You obviously can't call `this.nested/moduleStore`, so instead, you need to call `this.['nested/moduleStore']`.

Would it not be better if the naming convention was 'nestedModule' or if mapStores changed `nested/moduleStore` into `nestedModuleStore` automatically?

**Comment by posva** [comment]
done
